aten/src/ATen/core/Dict.h

```diff
@@ -196,6 +212,20 @@ class Dict final {
    */
   Dict();
 
+  /**
+   * Create a generic dict with runtime type information.
+   * This only works for c10::impl::GenericDict and is not part of the public API
+   * but only supposed to be used internally by PyTorch.
+   */
+  template<class _Key = Key, class _Value = Value>
+  explicit Dict(
```

Comment:
given that there's no other constructors - why can't it be just a static assert with a more human-readable message?